aspnetcore/migration/22-to-30.md

```diff
@@ -437,35 +437,45 @@ The most significant change from `WebHostBuilder` to `HostBuilder` is in [depend
 
 ## Update SignalR code
 
-If you call `AddJsonProtocol`, replace it with `AddNewtonsoftJsonProtocol`.
+System.Text.Json is now the default Hub Protocol used by both the client and server.
 
-* The following examples show server code before and after the change:
+Use `AddJsonProtocol` to set serializer options.
 
-  ```csharp
-  services.AddSignalR(...)
-          .AddJsonProtocol(...) // 2.2
-  ```
-
-  ```csharp
-  services.AddSignalR(...)
-          .AddNewtonsoftJsonProtocol(...) // 3.0
-  ```
+Server:
+```csharp
+services.AddSignalR(...)
+        .AddJsonProtocol(options =>
+        {
+            options.WriteIndented = false;
+        })
+```
 
-* The following examples show .NET client code before and after the change:
+Client:
+```csharp
+new HubConnectionBuilder()
+    .WithUrl("/chatHub")
+    .AddJsonProtocol(options =>
+    {
+        options.WriteIndented = false;
+    })
+    .Build();
+```
 
-  ```csharp
-  connection = new HubConnectionBuilder()
-      .WithUrl(...)
-      .AddJsonProtocol(...) // 2.2
-      .Build()
-  ```
+If you were previously using `AddJsonProtocol` with Newtonsoft.Json options or want to continue using Newtonsoft.Json instead of the new default the following instructions will show you how:
```

Comment:
This almost feels like it warrants a small heading to the effect of "Using Json.NET in SignalR" or "Continuing to use Json.NET in SignalR"